pkg/config/validate.go

```diff
@@ -282,6 +282,7 @@ func (dc *DeploymentConfig) getValidators() map[string]func(validatorConfig) err
 		testRegionExistsName.String():  dc.testRegionExists,
 		testZoneExistsName.String():    dc.testZoneExists,
 		testZoneInRegionName.String():  dc.testZoneInRegion,
+		testApisEnabledName.String():   dc.testApisEnabled,
```

Comment:
I don't think the ordering here has any effect but since we think we are going to run this validator first, to me it makes most sense to declare it first in the list.

Reply:
Done on `HEAD` of PR